### PR TITLE
issue/125 - remove Swagger UI due to vulnerabilities

### LINKED ISSUE #125: Remove Swagger UI due to vulnerabilities
Fixes #124

### PR DESCRIPTION
**Pull request by Monam-Bharti**
Fixes #125

### CODE REVIEW

File: ignition/templates/driver_template/docs/index.md [hunk 1]
```diff
@@ -35,8 +35,6 @@ Reference the values file on install to apply configuration:
 helm install --name {(helm.name)} {(helm.name)}-{(app.version)}.tgz -f custom_values.yaml-
 ```
 
-Once the pod for the driver has started you will be able to access the Swagger UI at: `http://<kubernetes-node-ip>:{(helm.node_port)}/api/driver/ui`
```
Comment (dvaccarosenna) [inline ignition/templates/driver_template/docs/index.md:38]: Can we replace this with the following:

"Once the pod for the driver has started you should be able to view the status of the application at: ` `http://<kubernetes-node-ip>:{(helm.node_port)}/management/health`"

This ensures the user has a way of confirming the app is running.

**Review by dvaccarosenna — CHANGES_REQUESTED** [review]

**Comment by dvaccarosenna** [comment]
Please create an issue in this repository for this change as #3485 refers to an internal tracking system which cannot be referenced here. We'll need an issue here to include in the release notes. 

FYI: https://github.com/IBM/ignition/blob/master/CONTRIBUTING.md#submitting-changes

**Comment by Monam-Bharti** [comment]
@dvaccarosenna Created a new issue #125  in the repository https://github.com/IBM/ignition .

File: ignition/templates/driver_template/docs/index.md (hunk 1 above)
Comment (Monam-Bharti) [reply ignition/templates/driver_template/docs/index.md:38]: Added the following as suggested:

Once the pod for the driver has started you should be able to view the status of the application at: `http://:{(helm.node_port)}/management/health`

Comment (Monam-Bharti) [reply ignition/templates/driver_template/docs/index.md:38]: @dvaccarosenna Please review again and let me know if any other change is needed.

Comment (dvaccarosenna) [reply ignition/templates/driver_template/docs/index.md:38]: Looks like Git reformatted by suggestion and removed part of it. 

The suggested URL should be: `http://<kubernetes-node-ip>:{(helm.node_port)}/management/health`

**Comment by dvaccarosenna** [comment]
As stated on the PR checks, the Travis build is failing. Looks like there are unit test failures, these will need to be resolved before the PR can be merged

File: ignition/templates/driver_template/docs/index.md (hunk 1 above)
Comment (Monam-Bharti) [reply ignition/templates/driver_template/docs/index.md:38]: Updated the correct URL

**Comment by Monam-Bharti** [comment]
Fixed the unit test failures. Travis build has passed.

**Review by chowarth123 — APPROVED** [review]

**Pull request merged**